include/graphblas/base/io.hpp

```diff
@@ -1078,6 +1078,72 @@ namespace grb {
 		return UNSUPPORTED;
 	}
 
+	/**
+	 * Sets all values of a matrix to the given value.
+	 *
+	 * Unmasked variant.
+	 *
+	 * @tparam descr     The descriptor used for this operation.
+	 * @tparam DataType  The type of each element in the given matrix.
+	 *                   Must not be void.
+	 * @tparam ValueType The type of the given value. Should be convertible
+	 *                   to \a DataType. Optional; default value is \a DataType.
+	 * @tparam backend   The backend that implements this function.
+	 *
+	 * \parblock
+	 * \par Accepted descriptors
+	 *   -# grb::descriptors::no_operation
+	 *   -# grb::descriptors::no_casting
+	 * \endparblock
+	 *
+	 * @param[in,out] C The vector of which every element is to be set to equal
+	 *                  \a val. On output, the number of elements shall be equal
+	 *                  to the size of \a C.
+	 * @param[in]   val The value to set each element of \a C to.
+	 * @param[in] phase Which #grb::Phase the operation is requested. Optional;
+	 *                  the default is #grb::EXECUTE.
+	 *
+	 * In #grb::RESIZE mode:
+	 * @returns #grb::SUCCESS  All the time, no de-/allocation is necessary.
```

Comment:
This is incorrect -- if there is a matrix C with capacity zero, then running this primitive in RESIZE mode must reallocate, is it not?